include/eve/module/core/regular/impl/simd/riscv/count_true.hpp

```diff
@@ -0,0 +1,39 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/concept/conditional.hpp>
+#include <eve/concept/value.hpp>
+#include <eve/detail/implementation.hpp>
+
+namespace eve::detail
+{
+template<scalar_value T, typename N, relative_conditional_expr C>
+EVE_FORCEINLINE std::ptrdiff_t
+                count_true_(EVE_SUPPORTS(cpu_), C cond, logical<wide<T, N>> v) noexcept
```

Comment:
You should check for `C::is_complete` and not call the masked versions for those.

(is_complete is ignore_all, ignore_none)
ignore_all - is_inverted == false
ignore_none - is_inverted == true